pkg/apis/pedia/v1alpha1/zz_generated.conversion.go

```diff
@@ -210,6 +212,8 @@ func autoConvert_pedia_ListOptions_To_v1alpha1_ListOptions(in *pedia.ListOptions
 		return err
 	}
 	// WARNING: in.OrderBy requires manual conversion: inconvertible types ([]github.com/clusterpedia-io/clusterpedia/pkg/apis/pedia.OrderBy vs string)
```

Comment:
There are also warnings, should be converted manually

Reply:
`autoConvert_pedia_ListOptions_To_v1alpha1_ListOptions` has been replaced with `Convert_pedia_ListOptions_To_v1alpha1_ListOptions` in`conversion.go`.
No suitable way found to prevent generating `autoConvert_pedia_ListOptions_To_v1alpha1_ListOptions`